controllers/dscinitialization/utils.go

```diff
@@ -440,3 +444,41 @@ func (r *DSCInitializationReconciler) createUserGroup(ctx context.Context, dscIn
 
 	return nil
 }
+
+func IsDefaultAudiences(specAudiences *[]string) bool {
+	return specAudiences == nil || reflect.DeepEqual(*specAudiences, defaultAudiences)
+}
+
+// GetEffectiveClusterAudiences returns the audiences defined for the cluster, falling back to the default audiences if needed.
```

Comment:
`if needed` - worth expanding what it actually means :)

Reply:
added another line! 
e349c3c